tests/tests_pytorch/strategies/test_colossalai.py

```diff
@@ -100,7 +100,7 @@ def test_gradient_clip_algorithm_error(tmpdir):
         trainer.fit(model)
 
 
-@RunIf(min_cuda_gpus=1, colossalai=True)
+@RunIf(min_cuda_gpus=1, standalone=True, colossalai=True)
```

Comment:
Why did you make this standalone? This just checks, whether appropriate error is raised, which is happening regardless of standalone.

Reply:
There is some deficiency in Colossalai distributed environment initialization. It assumes users using a consistent distributed environment in training. If standalone is False, the testing functions are launched in sequence. Releasing any `ColossalaiStrategy` instance leads to appropriate exist of distributed environment. This causes the next testing function can't initialize correctly. 
It is a problem that should be fixed by Colossalai. But we need to fix it in the next version. Currently, the `standalone` is necessary to be True.